_kratos/kratos.yml

```diff
@@ -114,7 +114,7 @@ hashers:
     key_length: 16
 
 identity:
-  default_schema_url: file:///etc/config/kratos/identity.schema.json
+  default_schema_url: file://./identity.schema.json
```

Comment:
@akshay196 You are not making use of this file right? Are we OK on using relative path here?

Reply:
Used in docker-compose.